real_estate/models/estate_property.py

```diff
@@ -0,0 +1,157 @@
+from odoo import api, fields, models
+from odoo.exceptions import UserError, ValidationError
+from odoo.tools.float_utils import float_compare
+from dateutil.relativedelta import relativedelta
+
+
+class EstateProperty(models.Model):
+    _name = 'estate.property'
+    _description = "Property"
+    _order = "id desc"
+
+    _sql_constraints = [
+        ("check_expected_price", "CHECK(expected_price > 0)", "Expected price must be"
+"positive."),
```

Comment:
```suggestion
        ("check_expected_price", "CHECK(expected_price > 0)", "Expected price must be positive."),
```
it's exactly 99 char